camel/utils/__init__.py

```diff
@@ -24,6 +24,8 @@
 from camel.messages import OpenAIMessage
 from camel.typing import ModelType, TaskType
 
+from .python_interpreter import PythonInterpreter, InterpreterError
+
 F = TypeVar('F', bound=Callable[..., Any])
 
 
```

Comment:
@lightaime we need the code gone from __init__.py apart from re-imports. Okay, maybe not in this MR.

Reply:
Sure!